### DISCUSSION TITLE
Possibility to model heterogenous graph with multiple edges between each pair of nodes

### DISCUSSION THREAD

**Discussion by stefanschutz** [pyg-team/pytorch_geometric#8633] [Q&A] [answered]
Hi, 

I have a use case that requires heterogenous graph where multiple edges are allowed between nodes. Following this document here: https://pytorch-geometric.readthedocs.io/en/latest/notes/heterogeneous.html, would it be possible to model the relations like:

```
["A", "relation_1", "B"]
["A", "relation_2", "B"]
["A", "relation_3", "B"]
...
```
From the example in the document, it seems this is not the case. So I wonder if I can model and perform GNN on such graphs for my use case. Thank you.

**Comment by rusty1s** [comment, marked as answer]
Yes, this is well supported.
